crates/language/src/python.rs

```diff
@@ -79,11 +79,22 @@ impl Language for Python {
 
     fn check_replacements(&self, n: NodeWithSource<'_>, replacements: &mut Vec<Replacement>) {
         if n.node.is_error() {
-            if n.text().is_ok_and(|t| t == "->") {
+            // https://github.com/getgrit/gritql/issues/416 single line example
+            if n.text().is_ok_and(|t| t == "->") || n.text().is_ok_and(|t| t == ",") {
                 replacements.push(Replacement::new(n.range(), ""));
             }
             return;
         }
+        if n.node.kind() == "," {
+            // https://github.com/getgrit/gritql/issues/416 multi line example
+            if let Some(prev) = n.node.prev_sibling() {
+                let empty = prev.range().end_byte() == prev.range().start_byte();
+                if prev.kind() == "identifier" && empty {
```

Comment:
Do we need the identifier restriction?

Reply:
good point, actually we don't: dcfcf4d0

that also passes all tests locally

I had gone down a few different rabbit holes / routes trying to make this work whilst ensuring all other tests passed as expected - one test that broke with one of my approaches was `test_notebooks::test_insertion` so I was trying to make it pretty strict but actually asserting that the previous node is empty is pretty strict and makes sense for this issue 👌

Reply:
Perfect, looks like this passes everything. I agree that empty is probably the best check.